### DISCUSSION TITLE
How to validate on initial display？

### DISCUSSION THREAD

**Discussion by WMinsane** [edmundhung/conform#664] [Q&A] [answered]
I want to validate default value of screen.
I use `shouldValidate: "onBlur"`  option so I tried code below and it works.
But It looks bad . I'd like to know how to do it better.
`  useEffect(() => {`  
`      form.validate()`  
`       setTimeout(() => {`  
`         inputRef.current?.focus();`  
`         inputRef.current?.blur();`  
`      }, 200);`  
`    }, []);`

**Comment by edmundhung** [comment]
you should be able to validate a field directly with `form.validate({ name: fields.something.name })`.

**Reply by WMinsane** [reply]
thank you for your replay.
Can that solution validate many fields?
I mean pass object and validate it in one line.

**Reply by edmundhung** [reply, marked as answer]
I forget that the name is optional on the validate intent. `form.validate()` should be enough if you wanna validate the whole form. You don't need to trigger any event from the inputs.

**Reply by WMinsane** [reply]
It worked! Thank you for your Quick response m(_ _)m
